### PR TITLE
Normalize mimetype

### PR DESCRIPTION
**Pull request by tmathern**
## Description

Normalize mimetypes to make sure they can be understood by other libraries. 

Fixes #CQ-4339915, [#ASSETS-347](https://jira.corp.adobe.com/browse/ASSETS-347)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: lib/utils/type.js [hunk 1]
```diff
@@ -19,50 +19,63 @@ const contentType = require('content-type');
...
-    try {
-        const { stdout } = await exec(`file -b --mime "${renditionPath}"`);
-
-        // parsing will throw if `file` command did not return a valid content type
-        const type = contentType.parse(stdout.trim());
-        result.mime = type.type || DEFAULT_MIME_TYPE;
-
-        if (type.parameters && type.parameters.charset) {
-            result.encoding = type.parameters.charset;
+class Mimetype {
```
Comment (jdelbick) [inline lib/utils/type.js:22]: I would recommend not refactoring the type.js file for this necessary fix but instead do it in a follow up PR

Comment (jdelbick) [reply lib/utils/type.js:22]: the main reason being, we export `detectContentType` so it would be a considered a breaking change

Comment (tmathern) [reply lib/utils/type.js:22]: Works with me.

**Comment by tmathern** [comment]
Replaced by https://github.com/adobe/asset-compute-pipeline/pull/44

**Pull request closed without merging**